addon/components/power-calendar-multiple.js

```diff
@@ -29,30 +30,38 @@ export default CalendarComponent.extend({
 
   // Actions
   actions: {
-    select(day, calendar, e) {
+    select(dayOrDays, calendar, e) {
       let action = this.get("onSelect");
+      let days;
+      console.log(dayOrDays)
+
+      if (Array.isArray(dayOrDays)) {
+        days = dayOrDays;
+      } else if (dayOrDays instanceof Object && dayOrDays.date instanceof Date) {
+        days = [dayOrDays];
+      } else {
+        assert(`The select action expects an array of date objects, or a date object. ${typeof dayOrDays} was recieved instead.`);
```

Comment:
Since assert is removed in production, this would leave an empty else branch.
The most idiomatic way of using assert would be at the very beginning of the function like this:
```js
select(dayOrDays, calendar, e) {
      assert(`The select action expects an array of date objects, or a date object. ${typeof dayOrDays} was recieved instead.`, Array.isArray(dayOrDays) || dayOrDays instanceof Object && dayOrDays.date instanceof Date)
  // the rest of the function can rely on days being correct.
}
```